src/commcare_cloud/commands/ansible/ansible_playbook.py

```diff
@@ -106,6 +108,17 @@ def get_limit():
             return ()
 
     def ansible_playbook(environment, playbook, *cmd_args):
+        if 'VIRTUAL_ENV' not in os.environ:
+            puts(color_notice("please activate the cchq virtualenv."))
+            return 2
```

Comment:
Is this assertion (that we're in a virtualenv) still necessary? I think the original motivation for adding it was because we had to use `pip` to get the ansible package version; since we aren't going to do that anymore, I think this will run fine not in a vitualenv as well. Typically this codebase doesn't require being in a virtualenv.

Reply:
@dannyroberts yes, we can remove it.